config/routes.rb

```diff
@@ -1,7 +1,11 @@
 Rails.application.routes.draw do
   get 'dashboard/index'
 
-  devise_for :users
```

Comment:
we need devise_for :users in order to use devise

Reply:
Ooops good catch! Fixing now.